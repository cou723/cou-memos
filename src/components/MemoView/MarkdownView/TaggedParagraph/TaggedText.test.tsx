/* eslint-disable @typescript-eslint/no-floating-promises */
// @vitest-environment jsdom

import { TaggedParagraph } from ".";
import { cleanup, render } from "@testing-library/react";
import { describe, afterEach, it, expect } from "vitest";

describe("TaggedText", () => {
    afterEach(cleanup);

    it("should render", () => {
        const { asFragment, getByText } = render(<TaggedParagraph text="#tag" />);
        expect(asFragment()).toMatchSnapshot();
        expect(getByText("#tag")).toBeTruthy();
    });
});
