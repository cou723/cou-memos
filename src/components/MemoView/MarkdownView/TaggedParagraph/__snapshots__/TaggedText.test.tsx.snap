// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TaggedText > should render 1`] = `
<DocumentFragment>
  <p />
  <div
    aria-label="Badge"
    class="badge mr-2 badge-neutral"
  >
    #tag
  </div>
  <p />
</DocumentFragment>
`;
